river/tree/extremely_fast_decision_tree.py

```diff
@@ -46,21 +47,14 @@ class ExtremelyFastDecisionTreeClassifier(HoeffdingTreeClassifier):
     nominal_attributes
         List of Nominal attributes identifiers. If empty, then assume that all numeric attributes
         should be treated as continuous.
-    attr_obs
-        The Attribute Observer (AO) used to monitor the class statistics of numeric
-        features and perform splits. Parameters can be passed to the AOs (when supported)
-        by using `attr_obs_params`. Valid options are:</br>
-        - `'bst'`: Binary Search Tree.</br>
-        - `'gaussian'`: Gaussian observer. The `n_splits` used to query
-         for split candidates can be adjusted (defaults to `10`).</br>
-        - `'histogram'`: Histogram-based class frequency estimation.  The number of histogram
-        bins (`n_bins` -- defaults to `256`) and the number of split point candidates to
-        evaluate (`n_splits` -- defaults to `32`) can be adjusted.</br>
-        See 'Notes' for more information about the AOs.
-    attr_obs_params
-        Parameters passed to the numeric AOs. See `attr_obs` for more information.
+    splitter
+        The Splitter or Attribute Observer (AO) used to monitor the class statistics of numeric
+        features and perform splits. Splitters are available in the `tree.splitter` module.
+        Different splitters are available for classification and regression tasks. Classification
+        and regression splitters can be distinguished by their property `is_target_class`.
+        By default, `tree.splitter.GaussianSplitter` is used if `splitter` is `None`.
```

Comment:
I would add: "This is an advanced option. Special care must be taken when choosing a different splitter." The same applies to other descriptions similar to this one.

Reply:
I added this comment in the last commit!